src/core/client/admin/components/StoryInfoDrawer/StoryInfoDrawer.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent } from "react";
+
+import { Card, Modal } from "coral-ui/components/v2";
+
+import StoryInfoDrawerQuery from "./StoryInfoDrawerQuery";
+
+import styles from "./StoryInfoDrawer.css";
+
+export interface StoryInfoDrawerProps {
+  open: boolean;
+  storyID?: string;
+  setStoryID?: (storyID: string) => void;
+  onClose: () => void;
+}
+
+const StoryInfoDrawer: FunctionComponent<StoryInfoDrawerProps> = ({
+  open,
+  onClose,
+  setStoryID,
+  storyID,
+}) => {
+  return (
+    <Modal open={open} onClose={onClose}>
+      {({ firstFocusableRef, lastFocusableRef }) => (
```

Comment:
I see a `firstFocusableRef` and a `lastFocusableRef`. Are they used anywhere, or is this copy pasted?